docs/getting-started-scalardb.md

```diff
@@ -0,0 +1,344 @@
+# Getting Started with Helm Charts (Scalar DB Server)
+
+This document explains how to get started with Scalar DB Server by using Helm Chart in your test environment. Here, we assume that you already have a Linux Server as a test environment.
```

Comment:
The guide should probably work with local Mac, right?
(We can let the QA team (Boney) see if it works with Mac.)

Reply:
Thank you for review!

> The guide should probably work with local Mac, right?

Yes. All component run as container on Docker.
And, all tools (Docker, minikube, kubectl, helm) provide binary for Mac.
So, probably, I think this guide works with local Mac.

Reply:
In the result of test by Boney-san, this guide works fine as expected in the Mac environment. 
So, I added `Mac` in the assumption environment as follows. 

> Here, we assume that you already have a Mac or Linux environment for testing.

1a087a9cabbc0f283ae00a33c38f82089f779d11